.github/workflows/container-build.yaml

```diff
@@ -0,0 +1,77 @@
+name: Create and publish Container image
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
```

Comment:
We don't use the `v` prefix for the tags in this repo -- https://github.com/meshtastic/python/tags

I imagine this should probably just build on any tag, or any three-part numeric thing that looks like a version number.